dogs.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adoptable Dogs</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <img class="header-img" src="images/adoptadog-logo.svg"  alt="Logo">
+      <h1 class="header-text">Seattle's premier dog adoption agency</h1>
+    </header>
+  <section class="navbar">
+    <nav>
+      <div class="nav">
+      <a class="header-links" href="index.html">Home</a>
+      <a>/</a>
+      <a class="header-links" href="dogs.html">Dogs</a>
+      <a>/</a>
+      <a class="header-links" href="blog.html">Blog</a>
+      </div>
+    </nav>
+  </section>
+
+
+    <h2> Here are all of our dogs!</h2>
+    <div class="container">
+      <div class="box">
+          <img class="card" src="images/murphy-card.jpg"  alt="Murphy Pic">
+          <h2>Murphy</h2>
+          <br><a class="pricing">Costs to Adopt:</a> $123.45
+          <br>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas velit risus, tempor eu neque vel, placerat eleifend diam. Pellentesque urna.
+          <input class="button" type=button onClick="parent.location='checkout.html'" value='Adopt'>
+      </div>
+      <div class="box">
+          <img class="card" src="images/poppy-card.jpg"  alt="Poppy Pic">
+          <h2>Poppy</h2>
+          <br><a class="pricing">Costs to Adopt:</a> $123.45
+          <br>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas velit risus, tempor eu neque vel, placerat eleifend diam. Pellentesque urna.
```

Comment:
Rather than these line breaks, perhaps you could just use margin to space things out?